openlibrary/templates/admin/sponsorship.html

```diff
@@ -59,8 +59,8 @@ <h1>Book Sponsorship</h1>
       <th>Total Unscanned Books</th>
       <td>$summary['total_unscanned_books']</td>
       $if ctx.user and ctx.user.is_admin():
-        $ isbns = ','.join([s['identifier'].replace('isbn_', '') for s in summary['books']])
-        <td><a href="https://8882k.csb.app/#isbns=$isbns">Print Slips</a></td>
+        $ isbns = ','.join([s['identifier'].replace('isbn_', '') for s in summary['books'] if s.get('repub_state') == '-1'])
+        <td><a href="https://8882k.csb.app/?isbns=$isbns">Print Slips</a></td>
```

Comment:
This link isn't working for me anymore; does it still work for you?

Reply:
I'm not sure if ever worked (exactly) w/ isbns added to it. But the `repub_state` is a useful change.